pom.xml

```diff
@@ -76,6 +76,12 @@
 				<artifactId>logback-json-classic</artifactId>
 				<version>0.1.5</version>
 			</dependency>
+
+			<dependency>
+				<groupId>com.google.guava</groupId>
+				<artifactId>guava</artifactId>
+				<version>20.0</version>
```

Comment:
I think the guava version is being set somewhere inside Spring to 18.0 and google-cloud-java requires 20.0.